requirements.txt

```diff
@@ -1,4 +1,5 @@
 alembic==0.9.6
+bcrypt==3.1.4
```

Comment:
You use `Flask-Bcrypt` module, but add only `bcrypt` as dependency.

Reply:
Flask-Bcrypt added